app/concepcts/reservations/use_cases/create.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+module Reservations
+  module UseCases
+    class Create
+      NotEnoughSeatsError = Class.new(StandardError)
+      def initialize(params:, repository: ReservationRepository.new, screening_repository: Screenings::ScreeningRepository.new)
+        @repository = repository
+        @screening_repository = screening_repository
+        @params = params
+        @screening = screening_repository.find(params[:screening_id])
```

Comment:
Consider that in this whole class you completely not using the `screening` instance. You just using ID of this instance. In this case, consider if you really have to have here access to the screenings repository and screening object. Maybe it's enough to use `params[:screening_id]` instead?